prompTraj demo

---

std_msgs/Bool success

